packages/providers/providers.yaml

```
@@ -10489,24 +10489,36 @@ ukg-pro:
     display_name: UKG Pro
     categories:
         - hr
-    auth_mode: API_KEY
+    auth_mode: BASIC
     proxy:
         base_url: https://${connectionConfig.hostname}
         headers:
-            authorization: Bearer ${apiKey}
+            content-type: application/json
+            us-customer-api-key: ${credentials.customerApiKey}
```

```suggestion
            us-customer-api-key: ${connectionConfig.customerApiKey}
```
We currently don't support interpolating credentials this way for the `BASIC` auth type.